libs/components/data-manager/src/lib/modules/data-manager/data-manager-column-picker/data-manager-column-picker.component.ts

```diff
@@ -96,6 +89,22 @@ export class SkyDataManagerColumnPickerComponent implements OnDestroy, OnInit {
     this._ngUnsubscribe.complete();
   }
 
+  public updateData(): void {
+    this.displayedColumnData = this.searchColumns(
+      this.columnData.filter((col) => !col.alwaysDisplayed)
+    );
+
+    if (this.dataState.onlyShowSelected) {
+      this.displayedColumnData = this.displayedColumnData.filter(
+        (col) => col.isSelected
+      );
+    }
+
+    this.isAnyDisplayedColumnSelected = this.displayedColumnData.some(
+      (aDisplayedColumn) => aDisplayedColumn.isSelected
```

Comment:
Change `aDisplayedColumn` to `col` to match the two functions above this one.

Reply:
fixed in commit: [rename](https://github.com/blackbaud/skyux/pull/349/commits/8e6bdadfec94e26f666d83e38fedad1d3b1ca731)